pkg/web/src/pages/browser/[[...paths]]/components/ProjectNameUpdate.tsx

```diff
@@ -2,17 +2,26 @@ import type { ProjectId } from '@violet/lib/types/branded'
 import { Loading } from '@violet/web/src/components/atoms/Loading'
 import { useApiContext } from '@violet/web/src/contexts/Api'
 import { useBrowserContext } from '@violet/web/src/contexts/Browser'
-import type { ChangeEvent, FormEvent } from 'react'
+import type { ChangeEvent, Dispatch, FormEvent } from 'react'
 import { useCallback, useEffect, useRef, useState } from 'react'
 import styled from 'styled-components'
 
 const InputFormProject = styled.form`
-  text-align: center;
+  text-align: left;
 `
 
-export const ProjectNameUpdate = (props: { confirmName: () => void; projectId: ProjectId }) => {
+export const ProjectNameUpdate = (props: {
+  confirmName: () => void
+  projectId: ProjectId
+  setNewProjectName: Dispatch<string>
+}) => {
   const [label, setLabel] = useState('')
-  const inputLabel = useCallback((e: ChangeEvent<HTMLInputElement>) => setLabel(e.target.value), [])
+  const inputLabel = useCallback(
+    (e: ChangeEvent<HTMLInputElement>) => {
+      setLabel(e.target.value), props.setNewProjectName(e.target.value)
+    },
+    [props]
```

Comment:
```suggestion
    [props.setNewProjectName]
```

基本的に最小限に依存させたほうがいいと思う (`api` はあれ一個で不変なのでいいと思う)